server/docs/source/index.html.md

```diff
@@ -286,15 +286,15 @@ follow | Stream logs
 
 ```json
 {
-	"id": "mygrid/misty-sun-87",
-	"node_id": "RQKP:Y32W:SB4H:7TNG:5BKC:R6ZO:5B25:C2AV:3Z3Q:SVPX:A76C:WPBX",
```

Comment:
Also considering adding a new `node_id ` field to the API, which just contains the same value as the current `id` field. It would be for future-compatibility: clients could start using the `node_id` field, and it would be easier to change the `id` field later.